Routes/backend/general.php

```diff
@@ -12,7 +12,7 @@ function () {
         //------------------------------------------------
         //------------------------------------------------
         Route::get( '/', 'PublicController@login' )
-            ->name( 'vh.backend' );
+            ->name( 'vh.backend' )->middleware('verify.auth');
```

Comment:
@we-vikram-wri231 is that a new middleware?

Reply:
@themodernpk Sir, I actually use this to verify security OTP. But that I already did in the model method. So I have removed it.